Configuration/HLT/python/autoHLT.py

```diff
@@ -3,9 +3,9 @@
 
 autoHLT = {
   'frozen25ns' : '25ns14e33_v1',
-  'relval25ns' : '25ns14e33_v1',
+  'relval25ns' : '25ns14e33_v3',
   'frozen50ns' : '50ns_5e33_v1',
-  'relval50ns' : '50ns_5e33_v1',
+  'relval50ns' : '50ns_5e33_v3',
   'fake'       : 'Fake',
```

Comment:
@Martin-Grunewald @fwyzard Right, as Andrea mentioned, it is just a temporary solution I change it here to get a compatible L1 menu, otherwise the workflows will just crash. I think you will put a separate PR similar to PR#11391 (75x) also to the 74x to update both autoCond.py and autoHLT.py to the most recent ones, am i right?